cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -336,67 +354,110 @@ func (cmd *command) convertToOverridesMap(key, value string) (string, map[string
 	return comp, latestOverrideMap, nil
 }
 
-func (cmd *command) showSuccessMessage() {
-	var err error
-	logger := cli.NewLogger(cmd.Verbose)
+//avoidUserInteraction returns true if user won't provide input
+func (cmd *command) avoidUserInteraction() bool {
+	return cmd.NonInteractive || cmd.CI
+}
 
-	fmt.Println("Kyma successfully installed.")
+func (cmd *command) printSummary(o deployment.Overrides) error {
```

Comment:
Maybe worth to move this whole output rending part to a template. What do you think?

Reply:
Definitely! Just moved the whole summary printing to the `nice` package as a first step towards this templating.